v2/component_analyses.yaml

```diff
@@ -0,0 +1,169 @@
+        schema:
+          type: string
+      - name: security-vendor
+        in: header
+        description: Security Vendor
+        schema:
+          type: string
+          enum:
+          - snyk
+
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/RegisteredUser'
+                    - $ref: '#/components/schemas/FreeUser'
+              examples:
+                RegisteredUser:
+                  value:
+                    recommendation:
+                      component_analyses:
+                        cve:
+                          - id: "CVE-2018-1002201"
+                            cvss: "1.2"
+
+                          - id: "CVE-2018-1002202"
+                            cvss: "1.3"
+                      recommended_version : 2
+                      message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+                      severity: "high"
+                      public_vulnerabilities_count: 2
+                      private_vulnerabilities_count: 1
+                      exploitable_vulnerabilities_count: 0
+                      vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+                FreeUser:
+                  value:
+                    recommendation:
+                      component_analyses:
+                        cve:
+                          - id: "CVE-2018-1002201"
+                            cvss: "1.2"
+
+                          - id: "CVE-2018-1002202"
+                            cvss: "1.3"
+                      recommended_version : 2
+                      message: "2 vulnerabilities within this dependency"
+                      severity: "high"
+                      public_vulnerabilities_count: 2
+                      vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+
+        202:
+          description: Package is currently Unavailable. Please try again later
+          content: {}
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+components:
+  schemas:
+    VulnerabilityDetails:
+      title: Vulnerability Details
+      type: object
+      properties:
+        vulneability:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+              cvss:
+                type: string
+      description: Information of the vulnerability found
+
+    RegisteredUser:
+      title: Vendor Private Recommendation Information
+      type: object
+      properties:
+        recommendation:
+          type: object
+          properties:
+            recommended_version:
```

Comment:
recommended_version type must be a string

Reply:
Ack!